From: Vincent ABRIOU <vincent.abriou@st.com>
Date: Tue, 11 Feb 2020 14:39:16 +0100
Subject: [PATCH 4/5] TFLite: tools: make: remove ruy tune_tool.cc from the
 build sources for libtensorflow-lite.a

The tune_tool.cc file should not be part of the libtensorflow-lite.a.
This file defines a main function that could not be embedded in the
libtensorflow-lite.a library.

Signed-off-by: Vincent ABRIOU <vincent.abriou@st.com>
---
 tensorflow/lite/tools/make/Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tensorflow/lite/tools/make/Makefile b/tensorflow/lite/tools/make/Makefile
index c1affd75e1..d221cbd45c 100644
--- a/tensorflow/lite/tools/make/Makefile
+++ b/tensorflow/lite/tools/make/Makefile
@@ -130,6 +130,7 @@ $(wildcard tensorflow/lite/*/*/*test.cc) \
 $(wildcard tensorflow/lite/*/*/*/*test.cc) \
 $(wildcard tensorflow/lite/kernels/*test_main.cc) \
 $(wildcard tensorflow/lite/kernels/*test_util*.cc) \
+$(wildcard tensorflow/lite/experimental/ruy/tune_tool.cc) \
 $(MINIMAL_SRCS)
 
 BUILD_WITH_MMAP ?= true
-- 
2.17.1
